### PR TITLE
DSP-777 Show message on all server errors in login component

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves [DSP-777](https://dasch.myjetbrains.com/youtrack/issue/DSP-777)

### CODE REVIEW

**Comment by mdelez** [comment]
can you give me a short explanation on how to test this please?

**Comment by kilchenmann** [comment]
@mdelez Sure. You can turn of the database for example: Stop the corresponding docker container. You can use the docker dashboard for this or the command line with `docker stop CONTAINER_ID`. The api should still run and will give you internal server error (status 500)

**Comment by kilchenmann** [comment]
When knora-stack is not running (no api, no db, no sipi, no salsah) you get a status 😳

**Comment by kilchenmann** [comment]
In case of a broken knora-stack (maybe only the api is running), you get a 503 error:

Screenshot 2020-10-05 at 11 27 26

File: projects/dsp-ui/src/lib/action/components/login-form/login-form.component.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class LoginFormComponent implements OnInit {
         forgot_pw: 'Forgot password?',
         error: {
             failed: 'Password or username is wrong',
-            server: 'There\'s an error with the server connection. Try it again later or inform the DaSCH team.'
+            server: 'There\'s an error with the server connection. Try it again later or contact the DaSCH support.'
```
Comment (mdelez) [inline projects/dsp-ui/src/lib/action/components/login-form/login-form.component.ts:81]: An error has occurred when connecting to the server. Try again later or contact the DaSCH support.

**Review by mdelez — CHANGES_REQUESTED** [review]
Just making the English a bit clearer

File: projects/dsp-ui/src/lib/action/components/login-form/login-form.component.ts (hunk 1 above)
Comment (kilchenmann) [reply projects/dsp-ui/src/lib/action/components/login-form/login-form.component.ts]: Done in b611048

**Review by mdelez — APPROVED** [review]

**Pull request merged**